src/stores/app/livekit-store.ts

```
@@ -119,7 +119,47 @@ export const useLiveKitStore = create<LiveKitState>((set, get) => ({
 
   requestPermissions: async () => {
     try {
-      if (Platform.OS === 'android' || Platform.OS === 'ios') {
+      if (Platform.OS === 'web') {
+        // On web, use the browser's MediaDevices API to request microphone permission
+        if (navigator.mediaDevices && navigator.mediaDevices.getUserMedia) {
+          try {
+            // Request microphone access - this will prompt the user for permission
+            const stream = await navigator.mediaDevices.getUserMedia({ audio: true });
+            // Release the stream immediately - LiveKit will get its own stream
+            stream.getTracks().forEach((track) => track.stop());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)